src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  switch (true) {
+    case (days >= 3 && days < 7):
+      return days * 40 - 20;
+
+    case (days >= 7):
+      return days * 40 - 50;
+
+    default:
+      return days * 40;
```

Comment:
`days * 40` - you use this expression in the several places, so makes sense to create a variable for that